.gitlab-ci.yml

```diff
@@ -15,6 +15,8 @@ before_script:
   - export HCA_CONFIG_FILE=~/.config/hca/config.$CI_COMMIT_REF_NAME.json
   - aws configure set default.s3.multipart_threshold 64MB
   - aws configure set default.s3.multipart_chunksize 64MB
+  - aws secretsmanager get-secret-value --secret-id dcp/dss/${CI_COMMIT_REF_NAME}/gcp-credentials.json | jq -r .SecretString
+  - export GOOGLE_APPLICATION_CREDENTIALS=$(pwd -P)/gcp-credentials.json
 
```

Comment:
I don't understand this code.  The output of `aws secretsmanager` is piped into `jq -r` which is going to output some results on stdout.  This output doesn't appear to be captured or redirected.  However then on the next line the export command assumes that a file `gcp-credentials.json` was created in the current directory.  How does this work?

Reply:
It doesn't work, thanks for catching this.

Fixed.